src/FluentValidation/Resources/Languages/SerbianLatinLanguage.cs

```diff
@@ -22,29 +22,29 @@
 
 namespace FluentValidation.Resources;
 
-internal class SerbianLanguage {
-	public const string Culture = "sr";
+internal class SerbianLatinLanguage {
+	public const string Culture = "sr-Latn";

```

Comment:
This can introduce breaking change, because `sr` culture will now be used on cyrillic version.